ReactWindows/ReactNative/ReactPage.cs

```diff
@@ -167,6 +167,17 @@ protected virtual ReactRootView CreateRootView()
         }
 
         /// <summary>
+        /// Called during OnCreate, to set the RootView's background
+        /// </summary>
+        /// <remarks>
+        /// Subclasses may override this method if it needs to use a custom background.
+        /// </remarks>
+        protected virtual void SetRootViewBackground()
+        {
```

Comment:
Actually another solution is to move `RootView.Background = ...` into `CreateRootView()`, so that I can override that method and return my own view with a custom Background. (If I did that now, my Background would be overridden in `onCreate`.)

So then I could do something like this in my app:

```c#
        protected override ReactRootView CreateRootView()
        {
            ReactRootView rootView = new ReactRootView();
            rootView.Background = new SolidColorBrush(ColorHelpers.Parse(0xFF123456));
            return rootView;
        }
``` 

That might be a better way to do it.